smith-daniels/minio
# Added threading to multiple item delete function for backblaze

## Description

Adds threading to deleting multiple objects in B2

## Motivation and Context

Veeam needs increased performance in order to not timeout

## How to test this PR?

Execute a large multiple object delete and confirm all files are deleted without errors

## Types of changes
- [x ] New feature (non-breaking change which adds functionality)
